### ISSUE TITLE
additional requirements

### ISSUE THREAD

**Issue by TeddJohnson** [highoncarbs/automato#25]
it may be helpful to add on that gcc and python3-devel (or distro equivalent) are needed prior to running `pip install -r requirements.txt`

**Comment by highoncarbs** [comment]
That would be on linux right ? I never had the problem on windows atleast . This usually happens when you'd build a library yourself .

**Comment by TeddJohnson** [comment]
yes, this is under Linux. I just wanted to add this as a note here at least, so that anyone else having problems would see it and find an easy solution :)

**Comment by highoncarbs** [comment]
Sure thanks for the input , would update the readme to reflect it .
